modules/cloud-guides/pages/ingest-catalog-data-teradata-s3-with-glue.adoc

```diff
@@ -0,0 +1,282 @@
+= Ingest and Catalog Data from Teradata Vantage to Amazon S3 with AWS Glue Scripts
+:experimental:
+:page-author: Daniel Herrera
+:page-email: daniel.herrera2@teradata.com
+:page-revdate: March 18, 2024
+:description: Ingest and catalog data from Teradata Vantage to Amazon S3
+:keywords: data warehouses, object storage, teradata, vantage, cloud data platform, data engineering, enterprise analytics, aws glue, aws lake formation, aws glue catalog 
+:dir: ingest-catalog-data-teradata-s3-with-glue
+
+== Overview
+This quickstart details the process of ingesting and cataloging data from Teradata Vantage to Amazon S3 with AWS Glue. 
+
+TIP: For ingesting data to Amazon S3 when cataloging is not a requirement consider https://docs.teradata.com/r/Enterprise_IntelliFlex_VMware/SQL-Data-Manipulation-Language/Working-with-External-Data/WRITE_NOS[Teradata Write NOS capabilities.]
+
+== Prerequisites
+* Access to an Amazon AWS account
```

Comment:
You can add  hyperlink pointing to https://aws.amazon.com/
`* Access to an https://aws.amazon.com/[Amazon AWS account]`